layouts/partials/home.html

```diff
@@ -1,5 +1,8 @@
 <section class="container centered">
   <div class="about">
+    {{ if .Site.Params.avatar }}
```

Comment:
You don't need the `avatar` config. Use `avatarurl` here. If it is missing, it will be false.